docs/src/ice_dynamics/ice_dynamics.md

```diff
@@ -165,7 +165,7 @@ In order to solve our equations, we will need numerical linear operators that gi
 ``` @example DEC
 function generate(sd, my_symbol; hodge=GeometricHodge())
   op = @match my_symbol begin
-    :♯ => x -> begin
+    :♯ᵖᵖ => x -> begin
       # This is an implementation of the "sharp" operator from the exterior
       # calculus, which takes co-vector fields to vector fields.
```

Comment:
Should we say that it "takes discrete vector fields to continuous vector fields"